README.md

```diff
@@ -12,9 +12,9 @@ that would normally come with a full Perl 6 distribution.  If you're
 after more than just the bare compiler, please download [the latest
 Rakudo Star package](http://rakudo.org/downloads/star).
 
-Note that different backends implement slightly different sets of
```

Comment:
Please return the part about backends implementing slightly different sets of features. This is actually true statement and a user must not expect all them to be running any kind of code equally well. Since there is no documentation page on this a plain text with no link would be ok.

Thank you!

Reply:
I have reinserted that note. I also found the [compilers page](https://perl6.org/compilers/) and now link to it instead of the FAQ.